src/components/structures/RoomView.js

```diff
@@ -266,6 +267,7 @@ module.exports = React.createClass({
                 console.log("Attempting to peek into room %s", roomId);
                 this.setState({
                     peekLoading: true,
+                    isPeeking: true, // this will change to false if peeking fails
```

Comment:
I don't think this comment is true

Reply:
@lukebarnard1 good catch, fixed.

Reply:
TBH, for symmetry I'd move `isPeeking: true` to line 275

Reply:
I started by doing that, and it effectively caused RRs to be sent on initial peek

Reply:
oh ok, fair enough!